README.md

```diff
@@ -32,7 +34,143 @@ For good first issues regarding contributing, see [issues labeled with open sour
 
 # Local development environment
 
-Please contribute by writing a document about local development environment. See above.
+This section is still under construction, if you have troubles setting the environment, please contact us (see below section).
+
+## Project structure
+
+The project is currently divided into three parts:
+
+- Backend, which that takes care of collecting and storing survey answers with a stong care for privacy and security
```

Comment:
```suggestion
- Backend, which takes care of collecting and storing survey responses with strong care for privacy and security
```

"responses" is the consistent terminology we try to use, for "answers" collected via the form. :wink: